.circleci/smoketest_docker.sh

```diff
@@ -145,6 +145,12 @@ echo "Uploader container ID == ${uploader_hostname}"
 echo "Using smoketest.py script from ${SMOKE_TESTS_SCRIPT_BRANCH} branch and URL ${SMOKE_TESTS_SCRIPT_URL}"
 
 function print_debugging_info_on_exit() {
+    echo ""
```

Comment:
If there isn't one already, could you make a ticket to move everything from CircleCI to Github Actions

Reply:
Good Idea. I will. CircleCI is not used anymore, just the name remains in the directory structure. Agent Build Refactored, dev-requirements-new.txt and so on will be good candidates for renaming to include in this ticket :)